.gitignore

```diff
@@ -29,3 +29,4 @@ web.*.config
 waca.sln.DotSettings.user
 .vs/*
 .arc*
+password.php
```

Comment:
I'm assuming your not going to add a file called password.php, if security through obscurity is your concern. :p